crates/sui-json-rpc/src/axum_router.rs

```diff
@@ -197,16 +199,27 @@ async fn handle_traffic_req(
     }
 }
 
+fn spam_weight_for_method(method: String) -> Weight {
+    // unless request requires gas payment, count against
+    // spam tally
+    match method.as_str() {
+        "sui_executeTransactionBlock" | "sui_devInspectTransactionBlock" => Weight::zero(),
```

Comment:
So, `sui_devInspectTransactionBlock` certainly could be spammed, and arguable a fullnode operator would want to count `sui_executeTransactionBlock` as spam as well, since its the validators that are being paid, not them.

Reply:
Done